client/dashboard/sites/overview/site-card.tsx

```diff
@@ -70,10 +74,16 @@ export default function SiteCard( {
 						<Field title={ __( 'Status' ) }>{ getSiteStatusLabel( site ) }</Field>
 					</HStack>
 					<HStack justify="space-between">
-						{ options?.software_version && (
-							<Field title={ __( 'WordPress' ) }>{ options.software_version }</Field>
+						<div style={ { flex: 1 } }>
+							<Field title={ __( 'WordPress' ) }>{ options?.software_version }</Field>
```

Comment:
Also we should keep the check for `options?.software_version`, no?

Reply:
Not sure why, I think we should anyway show the divs, maybe fall back to `?? 'Unknown'`

Reply:
Added a fallback, in case the REST API returns nothing

Reply:
I can't see how `Unknown` can be useful.. We'll just add noise to the UI without any good reason.

Reply:
Ok then what do you want to display? Nothing? Why should the field disappear when people are expecting it to be there?

Reply:
What's wrong with the existing way where we don't show the bit of information (i.e. WP version) when the backend does not return it? I think the logic is cleaner.

(P.S.: I'm not sure if there's actually a case where that `software_version` option is not set in the backend)

Reply:
>Ok then what do you want to display? Nothing? 

Exactly. Even if people expected that field to be there and we'd show `unknown`, it feels like a bug.

Reply:
Ok, I don't care strongly so reverted and added a condition for the whole section